src/components/ViewRequest/Details.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Accordion,
+  AccordionHeader,
+  AccordionItem,
+  AccordionPanel,
+  Label,
+  Text,
+  Title3,
+} from "@fluentui/react-components";
+import { useRequest } from "api/requestsApi";
+import { useParams } from "react-router-dom";
+
+const ViewRequestDetails = () => {
+  const params = useParams();
+  const request = useRequest(Number(params.requestId));
+
+  return (
+    <>
+      {request.data && (
```

Comment:
Because this component will always just be a child of the ViewRequest component, I believe it would be better to just pass down the RPARequest object fetched from the parent. No reason to create a RQ subscription b/c the parent's subscription will cause the child to rerender should the data change. Also it eliminates having to check if request.data is present as the parent already has the error/data checking present and will avoid loading this component altogether if no data is present. It will also make this component easier to test, because it is now simply an Input/Output check with no dependency on where the Input is coming from.